docs/source/reference/methods.rst

```diff
@@ -320,7 +372,7 @@ stop:
 
 
 streamlines:
-    full path to the complete, unsegmented tractography file
```

Comment:
I think this should actually be in kwargs.rst file

Reply:
Should have mentioned that these changes happened automatically when I ran `make html` to build the docs.